server.js

```diff
@@ -101,16 +102,19 @@ function parseLess(req) {
 }
 
 function outputCss(req, res) {
-   return function (parseTree) {
-      var css = parseTree.toCSS(req.parserOptions);
+   return function (css) {
       res.writeHead(200, "Content-Type: text/css");
       res.end(css);
-      return Q.fcall(function(){
-         return css;
-      });
+      return css;
    }
 }
 
+function renderCss(req, res) {
```

Comment:
Maybe order this definition above `outputCss` to reflect the usage ordering? No biggie though. Might even be outdated...